lib/rspec/core/formatters/helpers.rb

```diff
@@ -30,8 +30,8 @@ def self.format_duration(duration)
                       end
 
           if duration > 60
-            minutes = (duration.to_i / 60).to_i
-            seconds = duration - minutes * 60
+            minutes = (duration.round / 60).to_i
+            seconds = (duration - minutes * 60).abs
```

Comment:
But in that case isn't it just rounded off to 0?  When I see `abs` it makes me think that we're able to get values like `-7` and then the `abs` flips it to `7`...which doesn't make much sense.

Can we just use `.round` instead?  That would be less confusing if it works.